spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "projects show page" do
+  before :each do
+    @recycled_material_challenge = Challenge.create!(theme: "Recycled Material", project_budget: 1000)
+    @boardfit = @recycled_material_challenge.projects.create!(name: "Boardfit", material: "Cardboard Boxes")
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+  end
```

Comment:
You will also probably need to add some `ContestantProject` objects here in your setup as well so that you can associate projects with a contestant. Then you wouldn't need your `add_project` method.